app/src/androidTest/java/ch/epfl/qedit/util/LocationHelper.java

```diff
@@ -0,0 +1,34 @@
+package ch.epfl.qedit.util;
+
+import android.content.Context;
```

Comment:
I don't think it is a good idea to put that class in `util`.

Reply:
I didn't find a better place for now... If you have a suggestion, tell me ;).